src/main/resources/templates/error.html

```diff
@@ -13,6 +14,8 @@
   </style>
 </head>
 <body>
-<p>Sorry we ran into a error</p>
+<h3>Sorry we ran into a error</h3>
+<p  th:text="${error.title}"></p>
+<p th:text="${error.details}"></p>
 </body>
 </html>
```

Comment:
updated error page to show error details using thymeleaf tags.

Reply:
Careful here, as you don't want to give too many details to the user.
1) For an unsophisticated user, you are confusing them.
2) For a sophisticated user (hacker), you are arming them.